### ISSUE TITLE
Send automated email for the invited users

### ISSUE THREAD

**Issue by kaushikmacharla07** [SigNoz/signoz#3239]
Currently Admin needs to send invite link to each member manually, it would be better if an automated email can be sent to the invited users. Its also hard to send links manually to invited users during Onboarding.

**Comment by pranay01** [comment]
@ankitnayan Can we pick this up. Might be easy to do it for SaaS users - where we can run an SMTP server

**Comment by ankitnayan** [comment]
Yes...@makeavish can pick this up. A generic SMTP config which can be used for different things inside the code. Also, should this be paid or free in OSS?

**Comment by prashant-shahi** [comment]
Resolved by https://github.com/SigNoz/signoz/pull/4258

**Comment by prashant-shahi** [comment]
Support for SMTP service is merged.
It will be part of public release SigNoz v0.36.
